opencti-platform/opencti-front/src/private/components/data/DataTableToolBar.jsx

```diff
@@ -1392,23 +1393,25 @@ class DataTableToolBar extends Component {
           // endregion
           return (
             <>
-              <Toolbar style={{ minHeight: 40, marginTop: 2 }} data-testid='opencti-toolbar'>
-                <Typography
-                  className={classes.title}
-                  color="inherit"
-                  variant="subtitle1"
-                >
-                  <span
-                    style={{
-                      padding: '2px 5px 2px 5px',
-                      marginRight: 5,
-                      backgroundColor: theme.palette.secondary.main,
-                      color: theme.palette.secondary.contrastText,
-                    }}
+              <Toolbar style={{ minHeight: 40, marginTop: 2, display: 'flex', justifyContent: 'space-between', height: '100%' }} data-testid='opencti-toolbar'>
+                <Box sx={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
Just FYI for component `Box`, it is more efficient to use directly a `div` instead. `Box` is more pertinent than `div` when you want to use the MUI theme in its style